CMakeLists.txt

```diff
@@ -561,6 +561,9 @@ endif()
 set_source(xsimd)
 resolve_dependency(xsimd 10.0.0)
 
+set_source(fast_float)
+resolve_dependency(fast_float)
```

Comment:
Let's specify the minimum version `5.3.0` here.

Reply:
done